core/modules/wdb/QueryAction.cc

```diff
@@ -123,6 +125,7 @@ class QueryAction::Impl {
     wbase::Task::Ptr _task;
     std::shared_ptr<ChunkResourceMgr> _chunkResourceMgr;
     std::string _dbName;
+    std::string _hostName;
```

Comment:
If you do something like what I suggested below you probably don't need this data member, just use `::hostname()` everywhere.